### PR TITLE
Create use-reporting-in-dotnetcore-with-entity-framework-core.md

### PR DESCRIPTION
**Pull request by todorarabadzhiev**

### CODE REVIEW

File: knowledge-base/use-reporting-in-dotnetcore-with-entity-framework-core.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+			<td>Product</td>
+			<td>Progress® Telerik® Reporting</td>
+		</tr>
+	</tbody>
+</table>
+
+
+## Description
+The [EntityDataSource component](../entitydatasource) is dedicated to the Entity Framework and cannot work with Entity Framework Core, 
+although the latter can be used with .NET Framework 4.5+, .NET Core. Framework and .NET Standard -
```
Comment (praykov) [inline knowledge-base/use-reporting-in-dotnetcore-with-entity-framework-core.md]: the latter can be used with .NET Framework 4.5+, .NET Core. Framework and .NET Standard -

File: knowledge-base/use-reporting-in-dotnetcore-with-entity-framework-core.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+</table>
+
+
+## Description
+The [EntityDataSource component](../entitydatasource) is dedicated to the Entity Framework and cannot work with Entity Framework Core, 
+although the latter can be used with .NET Framework 4.5+, .NET Core. Framework and .NET Standard - 
+[EF Core Introduction](https://www.entityframeworktutorial.net/efcore/entity-framework-core.aspx).
+
+## Solution
+Use an **ObjectDataSource** component instead. In the latter, you may use the Entity Framework Core to fetch the data and return it
```
Comment (praykov) [inline knowledge-base/use-reporting-in-dotnetcore-with-entity-framework-core.md]: Use an **ObjectDataSource** component instead which supports the Entity Framework Core to fetch the data and return it

File: knowledge-base/use-reporting-in-dotnetcore-with-entity-framework-core.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+[EF Core Introduction](https://www.entityframeworktutorial.net/efcore/entity-framework-core.aspx).
+
+## Solution
+Use an **ObjectDataSource** component instead. In the latter, you may use the Entity Framework Core to fetch the data and return it 
+in one of the formats that the [ObjectDataSource](../objectdatasource) supports.
+
+If you would like to be able to use the same ObjectDataSource to design your reports in the 
+[Standalone Report Designer](../standalone-report-designer), the project should be built against **.NET Standard 2.0**. This technical 
+limitation comes from the fact that the Standalone Report Designer is a WPF application with Windows Forms Report Viewer built against 
+.NET Framework 4.6.2, and can resolve assemblies built with .NET Standard 2.0 - see
```
Comment (praykov) [inline knowledge-base/use-reporting-in-dotnetcore-with-entity-framework-core.md]: .NET Framework, and can resolve assemblies built with .NET Standard - see

File: knowledge-base/use-reporting-in-dotnetcore-with-entity-framework-core.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+Use an **ObjectDataSource** component instead. In the latter, you may use the Entity Framework Core to fetch the data and return it 
+in one of the formats that the [ObjectDataSource](../objectdatasource) supports.
+
+If you would like to be able to use the same ObjectDataSource to design your reports in the 
+[Standalone Report Designer](../standalone-report-designer), the project should be built against **.NET Standard 2.0**. This technical 
+limitation comes from the fact that the Standalone Report Designer is a WPF application with Windows Forms Report Viewer built against 
+.NET Framework 4.6.2, and can resolve assemblies built with .NET Standard 2.0 - see 
+[.NET implementation support](https://docs.microsoft.com/en-us/dotnet/standard/net-standard#net-implementation-support). The designer 
+cannot recognize assemblies that target .NET Core or .NET Standard 2.1. For that reason, the ObjectDataSource assembly should reference 
+.NET Standard 2.0. It will be necessary to register the assembly in the designer's configuration file as explained in the
```
Comment (praykov) [inline knowledge-base/use-reporting-in-dotnetcore-with-entity-framework-core.md]: .NET Standard 2.0. It is necessary to register the assembly in the designer's configuration file as explained in the

File: knowledge-base/use-reporting-in-dotnetcore-with-entity-framework-core.md [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+in one of the formats that the [ObjectDataSource](../objectdatasource) supports.
+
+If you would like to be able to use the same ObjectDataSource to design your reports in the 
+[Standalone Report Designer](../standalone-report-designer), the project should be built against **.NET Standard 2.0**. This technical 
+limitation comes from the fact that the Standalone Report Designer is a WPF application with Windows Forms Report Viewer built against 
+.NET Framework 4.6.2, and can resolve assemblies built with .NET Standard 2.0 - see 
+[.NET implementation support](https://docs.microsoft.com/en-us/dotnet/standard/net-standard#net-implementation-support). The designer 
+cannot recognize assemblies that target .NET Core or .NET Standard 2.1. For that reason, the ObjectDataSource assembly should reference 
+.NET Standard 2.0. It will be necessary to register the assembly in the designer's configuration file as explained in the 
+[Extending Report Designer](../standalone-report-designer-extending-configuration) article. It may be necessary to undertake also
```
Comment (KatGitHub) [inline knowledge-base/use-reporting-in-dotnetcore-with-entity-framework-core.md]: Words order should be as: "It also may be necessary to undertake the steps described in the.."

**Comment by KatGitHub** [comment]
My suggestion is to split the paragraph related to the Standalone designer into several logical parts. The paragraph is very large and it might be harder to read for some users.

**Pull request merged**